lib/scss/inc/_base.scss

```diff
@@ -1,6 +1,5 @@
 * {
   -moz-osx-font-smoothing: grayscale;
-  -webkit-font-smoothing: antialiased;
```

Comment:
Is there any reason behind removing this property? This was added for font smoothing in MacOS.

Reply:
We couldn't achieve the desired font thickness without removing this property, however if you think it might be a better idea we can add this back. CC: @deekoder

Reply:
@rushenn it was making the font look too thin. So we removed it. Are there any other side effects of removing this property in your opinion?